src/MonitorProfiler/Utilities/ILRewriter.cpp

```diff
@@ -600,195 +594,4 @@ void ILRewriter::DeallocateILMemory(LPBYTE pBody)
     }
 
     delete[] pBody;
-}
-
-
-/////////////////////////////////////////////////////////////////////////////////////////////////
-//
-// R E W R I T E
-//
-////////////////////////////////////////////////////////////////////////////////////////////////
-
-UINT ILRewriter::AddNewInt32Local()
-{
```

Comment:
Even if we are not using these methods, keeping them in is likely going to make future merges easier. Or did you need to take these out because they break compilation?